spec/classes/rabbitmq_spec.rb

```diff
@@ -6,7 +6,7 @@
     let(:facts) {{ :osfamily => 'Unsupported' }}
 
     it 'we fail' do
-      expect { subject }.to raise_error(/not supported on an Unsupported/)
+      expect { is_expected.to compile }.to raise_error(RSpec::Expectations::ExpectationNotMetError, /not supported on an Unsupported/)
```

Comment:
Could this be changed to just match the regex and not the error type? As in

```
expect { is_expected.to compile }.to raise_error(/not supported on an Unsupported/)
```

That way if they change how the error is detected again this will still work. Checking that the test framework raises an rspec error instead of checking that the catalog raises a puppet error seems strange to me.